Documentation/seccomp-guide.md

```diff
@@ -0,0 +1,375 @@
+# Seccomp Isolators Guide
+
+This document is a walk-through guide describing how to use rkt isolators for
+[Linux seccomp filtering](https://lwn.net/Articles/656307/).
+
+* [About Seccomp](#about-seccomp)
+* [Predefined Seccomp Filters](#predefined-seccomp-filter)
+* [Seccomp Isolators](#seccomp-isolators)
+* [Usage Example](#usage-example)
+* [Overriding Seccomp Filters](#overriding-seccomp-filters)
+* [Recommendations](#recommendations)
+
+## About seccomp
+
+Linux seccomp (short for SECure COMputing) filtering allows one to specify which
+system calls a process should be allowed to invoke, reducing the kernel surface
+exposed to applications.
+This provides a clearly defined mechanism to build sandboxed environments, where
+processes can run having access only to a specific reduced set of system calls.
+
+In the context of containers, seccomp filtering is useful for:
+
+* Restricting applications from invoking syscalls that can affect the host
+* Reducing kernel attack surface in case of security bugs
+
+For more details on how Linux seccomp filtering works, see
+[seccomp(2)](http://man7.org/linux/man-pages/man2/seccomp.2.html).
+
+## Predefined seccomp filters
+
+By default, rkt comes with a set of predefined filtering groups that can be
+used to quickly build sanboxed environments for containerized applications.
+These sets are just easier ways for referencing groups of syscalls covering
```

Comment:
Each set is simply a reference to a group of syscalls, covering a single functional area of kernel subsystem